docs/user/automation-rules.rst

```diff
@@ -64,50 +69,42 @@ Activate version
 
 Hide version
   Hides the version. If the version is not active, activates it and builds the version.
-  See :ref:`versions:Version States`.
+  See :ref:`versions:Version states`.
 
 Make version public
   Sets the version's privacy level to public.
-  See :ref:`versions:privacy levels`.
+  See :ref:`versions:Privacy levels`.
 
 Make version private
   Sets the version's privacy level to private.
-  See :ref:`versions:privacy levels`.
+  See :ref:`versions:Privacy levels`.
 
 Set version as default
-  Sets the version as default,
-  i.e. the version of your project that `/` redirects to.
-  See more in :ref:`automatic-redirects:Root URL`.
+  Sets the version as the :term:`default version`.
```

Comment:
I think we define this in another PR, but seemed useful :)

Reply:
YES! And PR has been merged :)